internal/config/array_flag.go

```diff
@@ -0,0 +1,14 @@
+package config
+
+import "strings"
+
+type ArrayFlags []string
+
+func (i *ArrayFlags) String() string {
+	return strings.Join(*i, ";")
+}
+
+func (i *ArrayFlags) Set(value string) error {
+	*i = append(*i, value)
+	return nil
+}
```

Comment:
the error is required because of the interface:
```
type Value interface {
	String() string
	Set(string) error
}
```
but i'll add/change to this:
```

var _ flag.Value = (*ArrayFlags)(nil)

//ArrayFlags implements the flag/Value interface
//allowing to set multiple string flags with the same name
type ArrayFlags []string
```